packages/react-app/components/notifications/OnboardingBatch.jsx

```diff
@@ -1,38 +1,133 @@
-import { Box, CloseButton, Flex, Text, Button, Link, Image, Spacer, Tooltip } from "@chakra-ui/react";
 import React from "react";
+import {
+  Box,
+  CloseButton,
+  Flex,
+  Text,
+  Button,
+  Link,
+  Image,
+  Spacer,
+  Tooltip,
+  useToast,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import { getUpdateBatchSignMessage, postUpdateBatch } from "../../data/api";
+import { useUserAddress } from "eth-hooks";
 
 const NEXT_BATCH_TELEGRAM = process.env.NEXT_PUBLIC_BATCH_TELEGRAM;
 const NEXT_BATCH_DATE = process.env.NEXT_PUBLIC_BATCH_DATE;
+const NEXT_BATCH = process.env.NEXT_PUBLIC_BATCH_NUMBER;
```

Comment:
I see that we don't have any of these env vars in the `.sample.env` file (inside NEXTJS).

This can be a good opportunity too add them (and clean the file if there are some unused stuff)